scripts/pre_push_hook.py

```diff
@@ -60,7 +61,11 @@
 FILE_DIR: Final = os.path.abspath(os.path.dirname(__file__))
 OPPIA_DIR: Final = os.path.join(FILE_DIR, os.pardir, os.pardir)
 LINTER_FILE_FLAG: Final = '--files'
-PYTHON_CMD: Final = 'python'
+
+# Path to currently running python interpreter,
+# it is required to resolve python version conflict in docker.
+PYTHON_CMD: Final = sys.executable if feconf.OPPIA_IS_DOCKERIZED else 'python'
```

Comment:
@gp201 Should we use `sys.executable` here instead, I agree with your POV, but if spreading of `OPPIA_IS_DOCKERIZED` is that important, then I would suggest using `sys.executable` instead of changing environment itself. Also found usage of `sys.executable` in many files (run_backend_tests, install_python_dev_dependencies, more)

Reply:
Can't we easily set `PYTHON_CMD` from the services.backend in docker-compose.yml file and use it here? (This will help us avoid changing the existing structure and also provide us with a scalable structure in the future i.e, configuring things for the container.)

Reply:
@DubeySandeep I didn't get it well. Do you mean changing the default Python in the docker environment, so that Python refers to `3.8.15`? If that's what you mean, it's already done, but I don't know why `python` refers to a different Python bin in Python's subprocess module. If you want we can change other bin (/usr/bin/python) to use 3.8.15.

python-bin used in python file /usr/bin/python
python-bin used by shell in docker /usr/local/bin/python